CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Dates are in `yyyy-mm-dd`.
 
 ### Changed
 * Login Screen design
+* Course row list item has a new, modern looking design
```

Comment:
The first part of this sentence ios too confusing. Reword it to: "_Changed_ Courses list for a new, modern looking design".